pynapple/core/ts_group.py

```diff
@@ -721,8 +704,8 @@ def count(self, bin_size=None, ep=None, time_units="s", dtype=None):
                 )[1]
 
             metadata = self._metadata.copy()
-            # drop rate from metadata
-            metadata = metadata.drop(columns=["rate"])
+            # drop rate
+            metadata.drop("rate")
```

Comment:
```suggestion
            metadata = metadata.drop(columns = ["rate"])
```
It's weird the test didn't catch it?